hermes/bin/src/runtime_extensions/wasi/clocks/state.rs

```diff
@@ -30,8 +30,8 @@ impl ClockState {
         let res_duration = std::time::Duration::from_nanos(1);
         let mono_resolution = match res_duration.as_nanos().try_into() {
             Ok(res) => res,
-            Err(_res_err) => {
-                // TODO(@saibatizoku): Log errors https://github.com/input-output-hk/hermes/issues/15
+            Err(res_err) => {
+                tracing::error!("Error converting duration to nanoseconds: {}", res_err);
```

Comment:
instead of putting the res_error inside the message, add it as a field:
```suggestion
                tracing::error!("Error converting duration to nanoseconds!", error= %res_err);
```